cfpb/rural-or-underserved-test
# Migrate cf-icons to SVG and fix search

## Changes
- Migrate the deprecated cf-icons to SVG icons.
- Fix the broken search input.

## Testing
- Follow building and running instructions https://github.com/cfpb/rural-or-underserved-test#building-and-viewing
(Run grunt with `./node_modules/grunt/bin/grunt` if it is no longer installed.)
- Check that the expandables on the main page FAQ work.
- Check that the search button expands.
- Enter an address and check that the map show/hide button works.

## Note
- I added some hack CSS to make some visual adjustments to the search input since it's a mix of old markup/CSS and new markup/CSS and won't matter when this is migrated completely into cfgov-refresh.

<img width="484" alt="Screen Shot 2019-05-16 at 3 29 38 PM" src="https://user-images.githubusercontent.com/704760/57881514-78422280-77ef-11e9-92d4-525a4d1467f0.png">
